lib/stripe/subscriptions/plan.ex

```diff
@@ -167,7 +173,8 @@ defmodule Stripe.Plan do
                optional(:ending_before) => t | Stripe.id(),
                optional(:limit) => 1..100,
                optional(:product) => Stripe.Product.t() | Stripe.id(),
-               optional(:starting_after) => t | Stripe.id()
+               optional(:starting_after) => t | Stripe.id(),
+               optional(:active) => boolean
```

Comment:
@snewcomer isn't this the change that you requested?

Reply:
I think github is having trouble displaying the diffs cleanly

Reply:
Oh you are so right.  Yep you added it!